jwk.go

```diff
@@ -87,6 +91,15 @@ func (k JsonWebKey) MarshalJSON() ([]byte, error) {
 	raw.Alg = k.Algorithm
 	raw.Use = k.Use
 
+	if len(k.Certificates) == 0 {
+		return MarshalJSON(raw)
+	}
+
+	raw.X5c = make([][]byte, len(k.Certificates))
+	for i, cert := range k.Certificates {
+		raw.X5c[i] = []byte(base64URLEncode(cert.Raw))
```

Comment:
This is the wrong encoding. Certificates needs to use base64 encoding, which is different from base64url encoding.